src/clojush/pushgp/report.clj

```diff
@@ -41,6 +41,9 @@
       (println (name param) "=" (random/seed-to-string val))
       (println (name param) "=" val))))
 
+(defn print-genome [individual]
+  (pr-str (not-lazy (map #(dissoc % :uuid :parent-uuid) (:genome individual)))))
```

Comment:
I realize that you inherited the `not-lazy` here, but I can't quite figure out why we need. I assume this is here because of @lspector's paranoia about laziness, but I don't see how laziness is an issue here. Since the string this generates in printed (immediately), that will force the list to be evaluated, right? Or is there something with threading that can act weirdly here?

Also, (and this isn't really an issue for this PR), this suggests that we might want some more configurability in specifying which fields you'd want to print in the reporting.